libsweep/src/protocol.cc

```diff
@@ -2,10 +2,15 @@
 #include <stdlib.h>
 
 #include "protocol.h"
+#include "../include/Error.hpp"
```

Comment:
Can do.
Although I'm really no fan of unqualified includes for files outside the same directory - in particular for files with very generic names.